app/src/main/java/ch/epfl/polychef/users/UserStorage.java

```diff
@@ -69,34 +89,77 @@ public void onCancelled(@NonNull DatabaseError databaseError) {
                 });
     }
 
-    private void initializeNewUser(String email) {
-        String username = getAuthenticatedUser().getDisplayName();
-        user = new User(email, username);
+    /**
+     * Get a {@code User} from an email.
```

Comment:
Maybe "Get a {@code User} by its email in Firebase database." would match the doc of `getUserByID` better

Reply:
👍